src/components/Friends/Friends.js

```diff
@@ -1,9 +1,40 @@
-import React from 'react';
+import React, { useState } from 'react';
+import FindPeople from './FindPeople';
+import { BottomNav } from "../BottomNav/BottomNav"
+import FriendsList from './FriendsList'
+import { Route } from 'react-router-dom';
 
-export default function Friends() {
+export default function Friends(props) {
+	console.log("props", props)
+	const [otherID, setOtherID] = useState('')
 
+	const handleID = (newID) => {
+		console.log("something", newID)
+		setOtherID(newID)
 
-	return <p> Friends</p>;
+		// console.log("e in friends.js", e)
+		props.onChange(newID)
+		// setOtherID()
+	}
+
+	// const passToParents = (e) => {
+	// 	console.log("e in passtoparents", e)
+	// 	// props.onChange(e)
+	// }
+	// onChange = { e => passToParents(e) }
+	return (
+		<React.Fragment>
+			<p> Friends</p>
+			<FindPeople name={otherID} onChange={handleID} />
+
+
+
+			<p> My friends</p>
+
+			<BottomNav />
```

Comment:
as long as you're navigating here from from the Dashboard, you don't need to include the BottomNav Component here because all you're doing it putting a second BottomNav Component on top of the first and it messes up the rendering of the onClick animation.

Reply:
So I removed the navbar as it was but cannot see it in the component. I am navigating from Friends.js and then findPeople.js. 
<img width="717" alt="Screen Shot 2020-11-18 at 12 45 23" src="https://user-images.githubusercontent.com/58600732/99526756-01658700-299c-11eb-8c95-c66c4da419be.png">